### ISSUE TITLE
Relic of Focus spams NREs on any stage without a teleporter

### ISSUE THREAD

**Issue by u-lulu** [elementGEMO/LunarsOfExiguity#3]
```
[04:01:31.6070364] [Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
LunarsOfExiguity.FocusConvergenceHooks+InvincibleDuringHoldout.FixedUpdate () (at <97ad59aa9b0e4dd48b13b2779a616cfb>:IL_0035)
```
This appears to have no effect on gameplay, but tends to bloat log size considerably.

**Comment by elementGEMO** [comment]
Unfortunately didn't see this before I published v0.1.2, but this seems easy to fix.

**Comment by elementGEMO** [comment]
I resolved the issue, but it'll have to wait to be released on either v0.1.3 or v.0.2.0, depending if I want to create more reworks and items first before the next update.
